vizro-core/src/vizro/static/css/aggrid.css

```diff
@@ -0,0 +1,53 @@
+.ag-theme-alpine.vizro {
+  --ag-background-color: var(--main-container-bg-color); /* confirmed */
+  --ag-foreground-color: var(--text-primary); /* confirmed */
+  --ag-odd-row-background-color: var(--main-container-bg-color); /* confirmed */
+  --ag-header-background-color: var(--main-container-bg-color); /* confirmed */
+  --ag-header-column-resize-handle-display: none; /* confirmed */
+  --ag-borders: none; /* confirmed */
+  --ag-icon-font-family: agGridBalham; /* confirmed, but hardly visible */
+  --ag-icon-size: 14px; /* confirmed */
+  --ag-row-height: 48px; /* This determines the row height, but not header */
+}
+/* General setting header row*/
+#page-container .ag-theme-alpine .ag-header-row {
+  color: var(--text-secondary);
+  font-weight: normal;
+  font-size: 14px;
+  height: 40px; /* this is not overwritten but has no effect */
```

Comment:
@AnnMarieW So I was under the impression that this is the handle, and it seems to not be overwritten, but at the same time changing it has no effect.

Reply:
The handle I was referring to was the one in the header used to adjust the width by dragging.  It's controlled by 

```
 --ag-header-column-resize-handle-display: none; /* confirmed */
```
When I take this out the handle shows up (it's too bright, but that can be fixed)

![image](https://github.com/mckinsey/vizro/assets/72614349/2d0519dd-df97-49da-8419-32cc548de14a)